pallets/loans/src/lib.rs

```diff
@@ -228,6 +234,18 @@ pub mod pallet {
         /// Event emitted when a market is activated
         /// [admin, asset_id]
         UpdatedMarket(Market<BalanceOf<T>>),
+        /// Reward added
+        RewardAdded(T::AccountId, BalanceOf<T>),
+        /// Reward withdrawed
+        RewardWithdrawn(T::AccountId, BalanceOf<T>),
+        /// Event emitted when market reward speed updated.
+        MarketRewardSpeedUpdated(AssetIdOf<T>, BalanceOf<T>),
+        /// Deposited when Reward is distributed to a supplier
+        DistributedSupplierReward(AssetIdOf<T>, T::AccountId, BalanceOf<T>, BalanceOf<T>),
```

Comment:
should put `Added`, `Updated` ..etc at the end. same to `UpdatedMarket`

Reply:
What do you mean by at the end. RewardAdded -> AddedReward?  @GopherJ